src/libs/connectorLibs/saveFiles.js

```diff
@@ -121,7 +125,7 @@ const saveEntry = async function (entry, options) {
         interval: 1000,
         throw_original: true,
         max_tries: options.retry,
-        args: [entry, options, method, file ? file._id : undefined]
+        args: [entry, options, client, method, file ? file._id : undefined]
```

Comment:
Do we really want the retry here? Having this kind of call with the need to list all the args is not very easy for the maintenance.

Reply:
Resolve and all prototype I change will be updated with client as 1st arg